unitary/alpha/quantum_world.py

```diff
@@ -111,9 +111,9 @@ def copy(self) -> "QuantumWorld":
         for remap in self.qubit_remapping_dict:
             new_dict = {}
             for key_obj, value_obj in remap.items():
-                new_dict[
-                    new_world.get_object_by_name(key_obj.name)
-                ] = new_world.get_object_by_name(value_obj.name)
+                new_dict[new_world.get_object_by_name(key_obj.name)] = (
+                    new_world.get_object_by_name(value_obj.name)
+                )
```

Comment:
What's up with all these formatting changes?  Are you on the same version of black as everyone else?

Reply:
There is a CI check on this repo that fails if the formatting is not identical to that which whichever black version the workflow installs produces, so I had to add a formatting commit to make it pass.

In the mean time, Peter already hit the same issue and merged a commit making those formatting changes, so now I can remove mine.

Reply:
ok, perfect.  I will merge this then.